hooli_data_eng/resources/__init__.py

```diff
@@ -0,0 +1,126 @@
+import os
+
+from dagster import EnvVar, FilesystemIOManager, ResourceDefinition
+from dagster._utils import file_relative_path
+from dagster_aws.s3 import ConfigurablePickledObjectS3IOManager, S3Resource
+from dagster_dbt import DbtCliClientResource
+from dagster_duckdb_pandas import DuckDBPandasIOManager
+from dagster_pyspark import pyspark_resource
+from dagster_snowflake_pandas import SnowflakePandasIOManager
+from dagstermill import ConfigurableLocalOutputNotebookIOManager
+
+from hooli_data_eng.resources.api import RawDataAPI
+from hooli_data_eng.resources.databricks import db_step_launcher
+from hooli_data_eng.resources.dbt import DbtCli2 as DbtCli
+#from hooli_data_eng.resources.warehouse import MySnowflakeIOManager as SnowflakePandasIOManager
+from hooli_data_eng.resources.sensor_file_managers import s3FileSystem, LocalFileSystem
+from hooli_data_eng.resources.sensor_smtp import LocalEmailAlert, SESEmailAlert
+
+
+# Resources represent external systems and, and specifically IO Managers
+# tell dagster where our assets should be materialized. In dagster
+# resources are separate from logical code to make it possible
+# to develop locally, run tests, and run integration tests
+#
+# This project is designed for everything to run locally
+# using the file system and DuckDB as the primary development resources
+#
+# PRs use a "branch" environment that mirrors production with
+# staging Snowflake and S3 resources
+#
+# The production deployment on Dagster Cloud uses production Snowflake
+# and S3 resources
+
+def get_env():
+    if os.getenv("DAGSTER_CLOUD_IS_BRANCH_DEPLOYMENT", "") == "1":
+        return "BRANCH"
+    if os.getenv("DAGSTER_CLOUD_DEPLOYMENT_NAME", "") == "data-eng-prod":
+        return "PROD"
+    return "LOCAL"
+
+
+# The dbt file dbt_project/config/profiles.yaml
+# specifies what databases to targets, and locally will
+# execute against a DuckDB
+
+DBT_PROJECT_DIR = file_relative_path(__file__, "../dbt_project")
```

Comment:
Are these relative paths still correct? I'd expect they need to be tweaked, though I'm not 100% sure

Reply:
I believe that is what is causing the branch deployment failure above

Reply:
I just fixed the paths, but haven't taken a look at the tests.  I'll get to those in a bit.